src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRefVcf.kt

```diff
@@ -27,7 +35,7 @@ class CreateRefVcf : CliktCommand(help = "Create a haplotype VCF for the referen
     val referenceUrl by option(help = "URL where the reference FASTA file can be downloaded")
         .default("")
 
-    val bed by option(help = "BED file with entries that define the haplotype boundaries")
+    val bed by option(help = "BED file with entries that define the haplotype boundaries .")
```

Comment:
I would remove this change.  Most of the other command help messages do not have periods at the end.

Reply:
yeah, this was introduced when I had another sentence here, then decided that sentence wasn't needed.  I'll remove the period.